backend/private-graph/graph/schema.resolvers.go

```diff
@@ -622,17 +622,19 @@ func (r *mutationResolver) EditWorkspace(ctx context.Context, id int, name *stri
 }
 
 // EditWorkspaceSettings is the resolver for the editWorkspaceSettings field.
-func (r *mutationResolver) EditWorkspaceSettings(ctx context.Context, workspaceID int, aiInsights *bool) (*model.AllWorkspaceSettings, error) {
+func (r *mutationResolver) EditWorkspaceSettings(ctx context.Context, workspaceID int, aiApplication *bool, aiInsights *bool) (*model.AllWorkspaceSettings, error) {
 	_, err := r.isAdminInWorkspace(ctx, workspaceID)
 	if err != nil {
 		return nil, err
 	}
-	workspaceSettings := &model.AllWorkspaceSettings{
-		WorkspaceID: workspaceID,
+
+	workspaceSettings := &model.AllWorkspaceSettings{}
+	workspaceSettingsUpdates := map[string]interface{}{
+		"AIApplication": *aiApplication,
+		"AIInsights":    *aiInsights,
 	}
-	if err := r.DB.Where(workspaceSettings).Updates(&model.AllWorkspaceSettings{
-		AIInsights: *aiInsights,
```

Comment:
Gorm won't update field to false, so refactored to fix bug
https://stackoverflow.com/questions/56653423/gorm-doesnt-update-boolean-field-to-false